src/ml_gym/frontend/dashboard/src/features/experimentConfig/experimentConfigSlice.tsx

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit'
```

Comment:
Codacy has a fix for the issue: [Replace `'@reduxjs/toolkit'` with `"@reduxjs/toolkit";`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=10020887)
```suggestion
import { createSlice } from "@reduxjs/toolkit";
```